sdk/lib/drivers/wdf/kmdf/src/librarycommon/fxlibrarycommon.cpp

```diff
@@ -480,6 +499,12 @@ FxLibraryCommonRegisterClient(
             __Print((LITERAL(WDF_LIBRARY_REGISTER_CLIENT)
                      ": Enhanced Verification is ON \n"));
 
+            __Print((LITERAL(WDF_LIBRARY_REGISTER_CLIENT)
+                     "Verificated functions table NOT IMPLEMENTED \n"));
```

Comment:
Move this before the first message and change to "WDF verification is not implemented"